DirectEmployers/MyJobs
# [ST-3728] The JavaScript for Django's select multiple expects a string, not an integer, for the id value

## What's this PR do?
Casts id to an string before returning it in the JSON response for the /admin/groupsites endpoint.

## Where should the reviewer start?
Check the Custom Facet admin page. Can you add SEO Site Facets without saving first?

## How should this be manually tested?
Create a Custom Facet. You should be able to add an SEO Site Facet without first saving the Custom Facet.

## Any background context you want to provide?
There may be other places this is broken that I wasn't aware of. It looks like they're all using the same code, so they should all be fixed as a result, but if you know that's not the case let me know so I can fix it.
